packages/teams-js/test/private/videoEffectsEx.spec.ts

```diff
@@ -40,6 +40,7 @@ describe('videoEffectsEx', () => {
         format: videoEffects.VideoFrameFormat.NV12,
         requireCameraStream: false,
         audioInferenceModel: new ArrayBuffer(100),
+        requiredCapabilities: [],
```

Comment:
I would expect to see a unit tests or unit tests that check the new attributes property (especially because the code that builds those attributes is very complex)